atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitionsTest.java

```diff
@@ -39,4 +43,20 @@ public void compactionStrategiesShouldMatchWithOrWithoutPackageName() {
                         onlyClassName.compaction_strategy),
                 ColumnFamilyDefinitions.isMatchingCf(fullyQualified, onlyClassName));
     }
+
+
+    @Test
+    public void ifCfDefsHaveDifferentGcGraceSecondsIsMatchingCfReturnsFalse() {
+        CfDef clientSideTable = ColumnFamilyDefinitions.getCfDef(
+                "test_keyspace",
+                TableReference.fromString("test_table"),
+                AtlasDbConstants.GENERIC_TABLE_METADATA);
```

Comment:
The aim here is to check if default tables are updates from the ONE_DAY setting.. renaming the test.